test/suite/test_tiered19.py

```diff
@@ -150,6 +150,25 @@ def test_ss_file_systems_gcp_and_azure(self):
         # Create another file systems to make sure that terminate works.
         ss.ss_customize_file_system(
             session, self.bucket, None, self.get_fs_config(prefix_2))
+
+        # We cannot use the file system to create files, it is readonly.
+        # So use python I/O to build up the file.
+        f = open('foobar', 'wb')
+
+        outbytes = ('MORE THAN ENOUGH DATA\n' * 100000).encode()
+        f.write(outbytes)
+        f.close()
+
+        # Flush valid file into Azure.
+        self.assertEqual(ss.ss_flush(session, azure_fs_1, 'foobar', 'foobar', None), 0)
+        # Check that file exists in Azure.
+        self.assertEqual(ss.ss_flush_finish(session, azure_fs_1, 'foobar', 'foobar', None), 0)
+
+        # Flush non valid file into Azure will result in an exception.
+        self.assertRaisesHavingMessage(wiredtiger.WiredTigerError,
+            lambda: ss.ss_flush(session, azure_fs_1, 'non_existing_file', 'non_existing_file', None), err_msg)
+        # Check that file does not exist in Azure.
+        self.assertEqual(ss.ss_flush_finish(session, azure_fs_1, 'non_existing_file', 'non_existing_file', None), 0)
```

Comment:
Add a comment to explain how this is different to the previous flush_finish

Reply:
added a comment https://github.com/wiredtiger/wiredtiger/pull/8763/commits/f070b3432d13428baf76c81c1474436bc3c4b921